app/auth/auth.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement, useState } from 'react';
+import { Alert, Button, StyleSheet, Text, View } from 'react-native';
+import Auth0 from 'react-native-auth0';
+import { AUTH0_CLIENT_ID, AUTH0_DOMAIN } from '@env';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#F5FCFF',
+  },
+  header: {
+    fontSize: 20,
+    textAlign: 'center',
+    margin: 10,
+  },
+});
+
+export const Auth = (): ReactElement => {
+  const env = {
+    clientId: AUTH0_CLIENT_ID,
+    domain: AUTH0_DOMAIN,
+  };
+  const auth0 = new Auth0(env);
+  const [accessToken, setAccessToken] = useState('');
+
+  /**
+   *
+   */
+  function onLogin(): void {
+    auth0.webAuth
+      .authorize({
+        scope: 'openid profile email',
+      })
+      .then((credentials) => {
```

Comment:
no, then / catch logic in the Views, we need to handle this with events and subscriptions